slash/log.py

```diff
@@ -106,6 +106,11 @@ def emit(self, record):
             reporter.notify_after_console_output()
         return returned
 
+def _slash_logs_filter(record, handler): # pylint: disable=unused-argument
+    return record.extra.get('filter_bypass') or \
+           record.level >= config.root.log.filter_log_level or \
+           not record.channel.startswith('slash')
```

Comment:
I think this should be a blacklist of channel names, not `startswith`. What if someone calls his or her logger `slashing` or `slash-infra`?